build.sbt

```diff
@@ -49,6 +53,8 @@ libraryDependencies ++= Seq(
   "eus.ixa" % "ixa-pipe-nerc" % "1.6.1"
 )
 
+libraryDependencies += "nz.ac.waikato.cms.weka" % "optics_dbScan" % "1.0.4"
```

Comment:
Oh man, what a blast from the past. I remember Weka from college ;) Would't recommend using it for any sort of non-toy data as their implementations are not well optimized: it's a teaching tool first and foremost, not a production tool.

As an alternative: I've heard good things about [ELKI](https://elki-project.github.io/releases/current/doc/de/lmu/ifi/dbs/elki/algorithm/clustering/DBSCAN.html).